src/molgenis/capice_resources/train_data_creator/data_parsers/clinvar.py

```diff
@@ -54,6 +54,10 @@ def _obtain_gene(self, clinvar_frame: pd.DataFrame) -> None:
                 Performed inplace.
         """
         self._process_column(clinvar_frame, TrainDataCreatorEnums.GENE.value, 'GENEINFO=')
+        # For now, deleting the SYMBOL ID as VKGL does not yet export this
+        clinvar_frame[
```

Comment:
GENEINFO can contain multiple genes:
```
##INFO=<ID=GENEINFO,Number=1,Type=String,Description="Gene(s) for the variant reported as gene symbol:gene id. The gene symbol and id are delimited by a colon (:) and each pair is delimited by a vertical bar (|)">
```

Example:
```
1       1474871 1295591 G       C       .       .       ALLELEID=1285386;CLNDISDB=MedGen:CN517202;CLNDN=not_provided;CLNHGVS=NC_000001.10:g.1474871G>C;CLNREVSTAT=criteria_provided,_single_submitter;CLNSIG=Benign;CLNVC=single_nucleotide_variant;CLNVCSO=SO:0001483;GENEINFO=TMEM240:339453|LOC121967044:121967044;MC=SO:0001627|intron_variant;ORIGIN=1
```

This happens for 121988 / 1578727 records (~8%) in recent GRCh38 ClinVar data. So I think CAPICE should take that into account.

Reply:
This complicates things quite a bid, as keeping both SYMBOL requires a more fundamental change in process-vep as well (which currently maps 1 to 1, after changes it would be 1 to many). Even if we switch to SYMBOL ID's.

Reply:
Discussed with @SietsmaRJ. Will fix later. Register issue in repo.